src/devtools/client/inspector/rules/models/element-style.js

```diff
@@ -86,11 +86,11 @@ class ElementStyle {
 
     this.onRefresh = this.onRefresh.bind(this);
 
-    if (this.ruleView.isNewRulesView) {
-      this.pageStyle.on("stylesheet-updated", this.onRefresh);
-      this.ruleView.inspector.styleChangeTracker.on("style-changed", this.onRefresh);
-      this.ruleView.selection.on("pseudoclass", this.onRefresh);
-    }
+    // if (this.ruleView.isNewRulesView) {
```

Comment:
should we keep this and pref it off via a new `edit` flag or just remove it in another pass and circle back?

Reply:
isNewRulesView isn't a pref. It's a parameter that only appears in new-rules.js to trigger new codepaths that was being implemented for the React-based Rules view. The goal was to move some of the business logic to the model where it belonged. 

In Replay, we don't have this.pageStyle anymore, so we will remove that probably in the future. The StyleChangeTracker and Pseudoclass might be important when we implement those features in Replay.